### PR TITLE
Make LICENSE detectable, add COPYRIGHT, and update file headers

### PR DESCRIPTION
**Pull request by opengeek**
### What does it do?
From @JoshuaLuckers in #14028:
> As a best practice the full license file should be placed in the root of the repository. It also makes it better detectable by automation tools.

In addition, I have renamed it to LICENSE since it is not in any necessary md format, added a COPYRIGHT file, and updated/added/fixed file headers throughout the codebase to reference both of these files.

### Why is it needed?
 * GitHub will be able to detect the license and people who visit the repository will see it at the top of the repository page.
 * This repository will show up when search results are being filtered based on their license.
 * Avoid repetitive updates to file headers.
 * Add missing file headers referencing the COPYRIGHT and LICENSE files.

### Related issue(s)/PR(s)
Supercedes #14028

### CODE REVIEW

**Review by JoshuaLuckers — DISMISSED** [review]

**Comment by Mark-H** [comment]
Well darnit, @Alroniks removes 2000 lines of code in #14034, and you add 9000 back! 😝

**Review by Mark-H — DISMISSED** [review]
I've reviewed as much of this as GitHub will let me :P

File: core/model/modx/modaccesspermission.class.php [hunk 1]
```diff
@@ -1,7 +1,13 @@
 <?php
-/**
- * @package modx
+/*
+ * This file is part of MODX Revolution.
+ *
+ * Copyright (c) MODX, LLC. All Rights Reserved.
+ *
+ * For complete copyright and license information, see the COPYRIGHT and LICENSE
+* files found in the top-level directory of this distribution.
```
Comment (Jako) [inline core/model/modx/modaccesspermission.class.php]: Just nitpicking: Why is that line not indented?
A lot of files below seems to have this issue.

**Review by Jako — COMMENTED** [review]
Maybe this could be improved (not critical).

File: core/model/modx/modaccesspermission.class.php (hunk 1 above)
Comment (opengeek) [reply core/model/modx/modaccesspermission.class.php]: There were a lot of regex global search and replaces going on. I could have messed up a few. Will search for and resolve that.

Comment (opengeek) [reply core/model/modx/modaccesspermission.class.php]: Resolved. Let me know if you see any other inconsistencies. Thanks!

**Review by Mark-H — APPROVED** [review]

**Review by alroniks — APPROVED** [review]
It seems OK to me. Merging!

**Pull request merged**